app/shared/resource-calendar/ResourceCalendar.js

```diff
@@ -22,10 +21,13 @@ export function UnconnectedResourceCalendar({
   selectedDate,
   t,
 }) {
+  const selectedDateArray = selectedDate.split('-');
```

Comment:
- [ ] This could be `const [year, month, dayNumber] = selectedDate.split('-');`

Then `selectedDay.setFullYear(selectedDateArray[0], selectedDateArray[1] - 1, selectedDateArray[2]);`
would become much more readable:
`selectedDay.setFullYear(year, month - 1, dayNumber);`.